### DISCUSSION TITLE
Delay in Accessing Academic Benefits After Verification?

### DISCUSSION THREAD

**Discussion by MohdIrfan14** [community/community#141619] [GitHub Education] [answered]
### Hi everyone,

I got a message saying my student status was verified on October 7, 2024, and that I don't need to reverify right now. But it also mentioned that there might be a wait before the academic benefits become available.

Has anyone else experienced this delay? I'm curious:

•How long does it usually take for the benefits 
 to kick in?
•Is there anything I should do while waiting?
•Do I get notified when the benefits are ready 
 to use?
Would love to hear your thoughts or experiences on this. Thanks!

**Comment by davevad93** [comment]
Hi @MohdIrfan14 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @MohdIrfan14 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @MohdIrfan14 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

**Comment by MohdIrfan14** [comment]
Hi Davide Presti,

Thank you so much for your detailed response and guidance! I really
appreciate you taking the time to explain the process and next steps.

Good news—my problem has been resolved, and I now have access to the
benefits. Your advice really helped me navigate the situation smoothly.

Thanks again for your support and for pointing me to the FAQs and
discussions. It made a big difference!

Best regards,
Mohd Irfan

**Reply by davevad93** [reply, marked as answer]
You're welcome @MohdIrfan14 , glad to help! 👍
